### ISSUE TITLE
Geographically Separated Scans:  How is Pogom handling this?

### ISSUE THREAD

**Issue by Illydth** [favll/pogom#398]
## Checklist
- [x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
- [x] I have checked if this issue already exists (both open and closed issues)
- [x] I am using the most recent version of pogom from the master branch
- [ ] If this is an installation issue, I am posting my OS, OS version, python version, pip version
- [ ] I'm posting the complete log
- [x] I'm using a descriptive title for this issue
## My Question:

For any of the programmers out there:  How is PogoM Handling a geographically separated set of scans?

For instance, lets say in my city I have 3 locations I want to scan:  My neighborhood and two parks that are approximately 30 - 40 miles away from my main scan location and at least 10 - 20 miles away from each other.

How is PogoM Handling this to get around Jump bans?  My guess is:  It's not, but I wanted to ask.  

Assuming the answer is "it's not", here's my proposal:

We all know Niantic has gotten aggressive with bans in the last month:  The moment they detect an account that has jumped from one location to another location in a time period that it could not have reached (I had 5 accounts scanning San Francisco and then brought them up in the MidWest when I flew back and got an account ban on all 5 within a few hours) they ban the account, upto and including a permanent ban.

If I'm scanning geographically separated regions with all accounts I have to assume the accounts are jumping from one location to another location to make the scan...which would account for a large portion of complaints about account bans on these boards.

My recommendation, then, is to allow POGOM to have accounts put into it and then "assigned" to scan areas:  When you drop a pin on the map you have a dropdown with a check box for all accounts POGOM knows about.  It then allows you to checkmark which accounts you want to use to scan that pin's area.  In this way, you can GUARANTEE that accounts aren't jumping from area to area and thus limit the number of bans on an account.

It's possible to do this same thing by starting multiple pogom's but that gets...difficult...to administrate.  Even in a Containerized solution you still have to build the container for each new POGOM you want to bring up (because the port is built into the container).

**Comment by iMagooo** [comment]
In all honesty I think this project is no longer maintained. I believe PokemonGo-Map has a pull request that aims to reduce the movement of accounts, probably worth checking out if interested.

**Comment by Illydth** [comment]
I don't think this is unmaintained, I've received at least a few replies from the maintainer.

That said, there is definitely a larger/more involved community over at PokemonGo Map.  The largest problem with that project is complete lack of a decent user interface and as far as I can tell a much higher rate of getting accounts banned due to exactly what I'm asking questions about here...though I have not run or delt with POGOMap at this point (considering it, so long as I can get it to load up in a docker container).

Really take the UI on this map and drop it into PogoMap and I think you'd have a damn nice tool.  That said, I'm actually pretty concerned about having to perpetually create accounts which get banned due to scan methodologies.  As this guy has run for well over 2 weeks without a problem on a single 1K area, I'm pretty sure it's safe, but I don't know with reports I read from POGOMap whether the same can be said for it.
